src/githubinfo.py

```diff
@@ -0,0 +1,67 @@
+platforms = [
```

Comment:
@kshitij10496 I suggest we save platforms and languages in a dictionary along with their respective  URL's and send that particular URL  to the user